src/avatar/examples.js

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import * as React from 'react';
+import {styled} from '../styles';
+import {Avatar} from './index';
+import examples from './examples-list';
+
+const Row = styled('div', {
+  width: '400px',
+  display: 'flex',
+  justifyContent: 'space-around',
+  alignItems: 'flex-end',
+  marginTop: '56px',
+});
+
+export default {
+  [examples.AVATAR_EXAMPLE]: function AvatarStory() {
+    return (
+      <Row>
+        {['scale800', 'scale1000', 'scale1200', 'scale1400'].map(
+          (size, index) => (
+            <Avatar
+              name={`user name # ${index}`}
+              size={size}
+              src={`https://api.adorable.io/avatars/285/${index}@adorable.io.png`}
```

Comment:
make sure we can use these icons under license

Reply:
Adorable Avatars is released under the terms of the MIT License.

Reply:
something this adorable must be shared with the world :)